sdk/ml/azure-ai-ml/azure/ai/ml/_local_endpoints/validators/environment_validator.py

```diff
@@ -83,7 +87,7 @@ def _get_cloud_environment_artifacts(
             blob_url=environment_asset.build.path,
             destination=download_path,
             datastore_operation=environment_operations._datastore_operation,
-            datastore_name=None,
+            datastore_name='workspaceartifactstore',
```

Comment:
What is this change?

Reply:
By default, the environment is searched in "workspaceblobstore" datastore. 
But when I tested, the environment is available in "workspaceartifactstore".